pkgs/by-name/ad/advcpmv/package.nix

```diff
@@ -0,0 +1,55 @@
+{ coreutils
+, fetchFromGitHub
+}:
+
+let
+  advcpmv-data = {
+    pname = "advcpmv";
+    patch-version = "0.9";
+    coreutils-version = "9.4";
+    version = "${advcpmv-data.patch-version}-${advcpmv-data.coreutils-version}";
+    src = fetchFromGitHub {
+      owner = "jarun";
+      repo = "advcpmv";
+      rev = "a1f8b505e691737db2f7f2b96275802c45f65c59";
+      hash = "sha256-IHfMu6PyGRPc87J/hbxMUdosmLq13K0oWa5fPLWKOvo=";
+    };
+    patch-file = advcpmv-data.src + "/advcpmv-${advcpmv-data.version}.patch";
+  };
+  coreutilsNoSingleBinary = coreutils.override { singleBinary = false; };
+in
+assert (advcpmv-data.coreutils-version == coreutils.version);
```

Comment:
Add spaces around this assert, easier to read

Reply:
Explain.

Reply:
Just a nitpick actually